### PR TITLE
Adjust version detection for Liberty and Mitaka as well.

### PR DESCRIPTION
**Pull request by joshw**
The nova versioning scheme change occurred earlier than I'd thought.

### CODE REVIEW

File: ZenPacks/zenoss/OpenStackInfrastructure/modeler/plugins/zenoss/cmd/linux/openstack/nova.py [hunk 1]
```diff
@@ -34,11 +35,12 @@ def get_productKey(version):
         ('2014.2', version),   # Juno
         ('2015.1', version),   # Kilo
         ('2015.2', version),   # Liberty
```
Comment (eimajenthat) [inline ZenPacks/zenoss/OpenStackInfrastructure/modeler/plugins/zenoss/cmd/linux/openstack/nova.py]: Should this Liberty line be removed as well?

**Comment by eimajenthat** [comment]
Had a question.

File: ZenPacks/zenoss/OpenStackInfrastructure/modeler/plugins/zenoss/cmd/linux/openstack/nova.py (hunk 1 above)
Comment (joshw) [inline ZenPacks/zenoss/OpenStackInfrastructure/modeler/plugins/zenoss/cmd/linux/openstack/nova.py]: nope, that's intentional- i didn't have a liberty system handy to see which way it returns its version, so i included both possibilities.

Comment (eimajenthat) [inline ZenPacks/zenoss/OpenStackInfrastructure/modeler/plugins/zenoss/cmd/linux/openstack/nova.py]: Per Josh, is intentional.  Just wanted to be sure

**Pull request merged**